.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  helm-unit-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Install Helm
```

Comment:
[dust] I'm coming at this with completely new eyes, but the `curl` made me wonder if there's an existing action that could install helm. Would the following work?

```yaml
     -  uses: azure/setup-helm@v4.2.0
        with:
          version: v3.14.4
```

Reply:
BTW, I got this from https://github.com/helm/chart-testing-action

Reply:
I've updated the step to use the install helm action from azure!